examples/finder_email_signup/frontend/cma-cases-email-signup.json

```diff
@@ -76,6 +76,9 @@
   "description": "You'll get an email each time a case is updated or a new case is published.",
   "details": {
     "beta": false,
+    "filter": {
+      "document_type": "cma_case"
```

Comment:
Should this example use `format`?

Reply:
They should be, but the tests in finder-frontend depend on these examples so will fail. I'll think about how to change the finder-frontend tests so they're more flexible to schema changes like this.